tests/test_utils.py

```diff
@@ -30,14 +30,18 @@ def test_find_file(tmp_path: Path) -> None:
 
 
 def test_store_results() -> None:
-    results = {"example_name": "example_value", "other_name": "other_value"}
+    results = {
+        "example_name": "example_value",
+        "other_name": "other_value",
+        "none": None,
```

Comment:
Does this actually work? I just tried on my python3 terminal, and file.write(None) throws an exception.

Reply:
Good point, but that's what we test here! In the store_results function body: 
```
        if result_value is None:
            result_value = ""
```